doc/classes/CPUParticles2D.xml

```diff
@@ -68,132 +93,194 @@
 		<member name="gravity" type="Vector2" setter="set_gravity" getter="get_gravity">
+			Gravity applied to every particle. Default value: [code](0, 98)[/code].
 		</member>
 		<member name="hue_variation" type="float" setter="set_param" getter="get_param">
+			Initial hue variation applied to each particle.
 		</member>
 		<member name="hue_variation_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's hue will vary along this [Curve].
 		</member>
 		<member name="hue_variation_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Hue variation randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="initial_velocity" type="float" setter="set_param" getter="get_param">
+			Initial velocity magnitude for each particle. Direction comes from [member spread] and the node's orientation.
 		</member>
 		<member name="initial_velocity_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Initial velocity randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="lifetime" type="float" setter="set_lifetime" getter="get_lifetime">
+			Amount of time each particle will exist. Default value: [code]1[/code].
 		</member>
 		<member name="linear_accel" type="float" setter="set_param" getter="get_param">
+			Linear acceleration applied to each particle in the direction of motion.
 		</member>
 		<member name="linear_accel_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's linear acceleration will vary along this [CurveTexture].
 		</member>
 		<member name="linear_accel_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+			Linear acceleration randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="local_coords" type="bool" setter="set_use_local_coordinates" getter="get_use_local_coordinates">
+			If [code]true[/code], particles use the parent node's coordinate space. If [code]false[/code], they use global coordinates. Default value: [code]true[/code].
 		</member>
 		<member name="normalmap" type="Texture" setter="set_normalmap" getter="get_normalmap">
+			Normal map to be used for the [code]texture[/code] property.
 		</member>
 		<member name="one_shot" type="bool" setter="set_one_shot" getter="get_one_shot">
+			If [code]true[/code], only one emission cycle occurs. If set [code]true[/code] during a cycle, emission will stop at the cycle's end. Default value: [code]false[/code].
 		</member>
 		<member name="orbit_velocity" type="float" setter="set_param" getter="get_param">
+			Orbital velocity applied to each particle. Makes the particles circle around origin. Specified in number of full rotations around origin per second.
 		</member>
 		<member name="orbit_velocity_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+			Each particle's orbital velocity will vary along this [CurveTexture].
```

Comment:
You fixed others but missed this one, it should be `Curve`.

Reply:
Sorry - I clearly rushed through this faster than I should have. Corrections incoming.